torch/nn/attention/flex_attention.py

```diff
@@ -1548,6 +1578,17 @@ def score_mod(
         else:
             return out
 
+    if not _FLEX_ATTENTION_DISABLE_COMPILE_DEBUG:
+        _warn_once(
+            "flex_attention_performance",
+            "flex_attention called without torch.compile() - this will use an unrolled vmap "
```

Comment:
Just say "this will use an unfused implementation that ... instead of generating a fused kernel"

Not just breakpoints, you can also put prints in there. Just say "if you want to debug your score_mod/mask_mod, you can set <x> which will allow you to print print statements or breakpoints"

For not work, specifically say that it doesn't work with the backwards pass.